include/stdarg.h

```diff
@@ -0,0 +1,205 @@
+/* stdarg.h for GNU.
+   Note that the type used in va_arg is supposed to match the
+   actual type **after default promotions**.
+   Thus, va_arg (..., short) is not valid.  */
```

Comment:
Should probably have the GPL license here if this is copied from GCC sources

Reply:
Also can you elaborate on "after default promotions"?

Reply:
no clue what that means, I just included this from our compiler's source.

Reply:
oh lol